packages/@aws-cdk/aws-ec2/lib/vpc.ts

```diff
@@ -376,6 +418,13 @@ abstract class VpcBase extends Resource implements IVpc {
     });
   }
 
+  /**
+   * Returns the Id of the VPN Gateway Id if any, or Null
+   */
+  public obtainVpnGatewayId(): string | undefined {
```

Comment:
```suggestion
  public get vpnGatewayId(): string | undefined {
```

Reply:
I'm then getting the error: `Duplicate Identifier 'vpnGatewayId'`. I'm aware that I can call it something else, I just don't feel comfortable renaming it when the variable returned is called vpnGatewayId. Any ideas?

Reply:
You should be calling it `public get vpnGatewayId`. I'm curious to see what it conflicts with, but that other thing is probably a field (defined where?) and should then be removed.

Reply:
Done. Added a getter and a setter, then removed the old vpnGatewayId from VpcBase. The approach looks a bit complicated to me, but I'm a typescript noob, so.